docker/Dockerfile.dev.api

```
@@ -3,7 +3,8 @@ FROM ruby:3.2.0-buster
 
 # 開発環境では `rails credentials:edit` で使うテキストエディタが必要になるので vim をインストール
 # ローカルに rails をインストールしてる場合は好きなエディタを用いてもよい
-RUN apt-get update && apt-get install -y vim
+RUN apt-get update && apt-get install -y vim \
```

```suggestion
RUN apt-get update && apt-get install -y \
    # コメントここに移す
    vim \
```

この変更でコメントの指してる位置に`default-mysql-client`も含まれてしまうので、こんな感じでコメントの指しているものが明確になるようにしてもいいかもしれません。

---

なるほど、そのような書き方ができるのですね！
これは知らなかったので勉強になりました！